packages/faustwp-core/package.json

```diff
@@ -54,8 +65,9 @@
     "test": "jest",
     "build-cjs": "tsc -p tsconfig-cjs.json",
     "build-js": "tsc -p .",
-    "build-styles": "sass src/styles/admin-bar.scss dist/css/admin-bar.css --embed-sources",
-    "watch-styles": "sass src/styles/admin-bar.scss dist/css/admin-bar.css --embed-sources --watch",
+    "build-fonts": "mkdir -p dist/fonts && shx cp src/fonts/* dist/fonts",
```

Comment:
Due to the way exports works, the dashicon font from WordPress core needs to be copied into the dist directory.